WebRoot/messages/ZaMsg_ja.properties

```
@@ -1779,7 +1779,7 @@ ERROR_INVALID_VOLUME_PATH = \u6307\u5b9a\u3057\u305f\u30dc\u30ea\u30e5\u30fc\u30
 ERROR_NO_SUCH_VOLUME = \u6307\u5b9a\u3057\u305f\u30dc\u30ea\u30e5\u30fc\u30e0\u306f\u5b58\u5728\u3057\u307e\u305b\u3093\uff01
 ERROR_VOLUME_ALREADY_EXISTS = \u6307\u5b9a\u3057\u305f\u30dc\u30ea\u30e5\u30fc\u30e0\u306f\u3059\u3067\u306b\u5b58\u5728\u3057\u307e\u3059\uff01
 ERROR_MAX_ACCOUNT_QUOTA = \u30a2\u30ab\u30a6\u30f3\u30c8\u306e\u5272\u308a\u5f53\u3066\u5bb9\u91cf\u306e\u5024\u304c\u7121\u52b9\u3067\u3059\u3002\u30a2\u30ab\u30a6\u30f3\u30c8\u306e\u5272\u308a\u5f53\u3066\u5bb9\u91cf\u304c\u3001\u30c9\u30e1\u30a4\u30f3\u7ba1\u7406\u8005\u304c\uff11\u4eba\u306e\u30e6\u30fc\u30b6\u30fc\u306b\u5bfe\u3057\u3066\u8a2d\u5b9a\u3067\u304d\u308b\u30e1\u30fc\u30eb\u306e\u6700\u5927\u5272\u308a\u5f53\u3066\u5bb9\u91cf\u3092\u8d85\u3048\u3066\u3044\u307e\u3059\u3002
-ERROR_MAX_ATTCHMENT_SIZE = \u6dfb\u4ed8\u30d5\u30a1\u30a4\u30eb\u306e\u6700\u5927\u30b5\u30a4\u30ba\u304c\u3001\u73fe\u5728 {0} MB \u306b\u8a2d\u5b9a\u3055\u308c\u3066\u3044\u308b\u6700\u5927\u30e1\u30c3\u30bb\u30fc\u30b8\u30b5\u30a4\u30ba\u3092\u8d85\u3048\u3066\u3044\u307e\u3059\u3002
+ERROR_MAX_MAIL_SIZE = \u30e1\u30fc\u30eb \u30b5\u30a4\u30ba\u304c\u3001\u73fe\u5728 {0} MB \u306b\u8a2d\u5b9a\u3055\u308c\u3066\u3044\u308b\u6700\u5927\u30e1\u30c3\u30bb\u30fc\u30b8 \u30b5\u30a4\u30ba\u3092\u8d85\u3048\u3066\u3044\u307e\u3059\u3002
```

Please remove two white spaces
```suggestion
ERROR_MAX_MAIL_SIZE = \u30e1\u30fc\u30eb \u30b5\u30a4\u30ba\u304c\u3001\u73fe\u5728 {0} MB \u306b\u8a2d\u5b9a\u3055\u308c\u3066\u3044\u308b\u6700\u5927\u30e1\u30c3\u30bb\u30fc\u30b8 \u30b5\u30a4\u30ba\u3092\u8d85\u3048\u3066\u3044\u307e\u3059\u3002
```
```suggestion
ERROR_MAX_MAIL_SIZE = \u30e1\u30fc\u30eb\u30b5\u30a4\u30ba\u304c\u3001\u73fe\u5728 {0} MB \u306b\u8a2d\u5b9a\u3055\u308c\u3066\u3044\u308b\u6700\u5927\u30e1\u30c3\u30bb\u30fc\u30b8\u30b5\u30a4\u30ba\u3092\u8d85\u3048\u3066\u3044\u307e\u3059\u3002
```

---

Updated.

---

Thanks!